framework/test/unit/engine/consensus/certificate_generation/commit_pool.spec.ts

```diff
@@ -241,7 +235,91 @@ describe('CommitPool', () => {
 			expect(commitPool['_gossipedCommits'].getByHeight(1070)).toBeArrayOfSize(0);
 		});
 
-		it('should select non gossiped commits that are created by the generator of the engine', async () => {
+		it(`should not clean commits for the heights ${maxHeightPrecommitted} - ${COMMIT_RANGE_STORED} until currentHeight and bftParams does not exist for height`, async () => {
+			// // Update current height so that commits will always be in the future
+			chain.lastBlock = { header: { height: 1019 } };
```

Comment:
This value can also be deduced from `maxHeightPrecommitted and COMMIT_RANGE_STORED`